Hearthstone Deck Tracker/Hearthstone/Player.cs

```diff
@@ -231,6 +236,19 @@ public void Play(Entity entity, int turn)
 				case (int)CardType.TOKEN:
 					entity.Info.Created = true;
 					break;
+				case (int)CardType.MINION:
+					if (entity.Card.Name.Equals("Pogo-Hopper"))
+						{
+							if(_game.OpponentEntity.IsCurrentPlayer)
```

Comment:
This is the player class, this is always either the local or the opposing player. `Player.Play()` is only called with entities for that player.